scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -0,0 +1,145 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+import "GvsExtractAvroFilesForHail.wdl" as ExtractAvroFilesForHail
+import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
+
+workflow GvsQuickstartHailIntegration {
+    input {
+        String branch_name
+        String hail_wheel = "gs://gvs-internal-scratch/hail-wheels/2022-10-18/0.2.102-964bee061eb0/hail-0.2.102-py3-none-any.whl"
+    }
+
+    String project_id = "gvs-internal"
+
+    call QuickstartVcfIntegration.GvsQuickstartVcfIntegration {
+        input:
+            branch_name = branch_name,
+            drop_state = "NONE",
+            extract_do_not_filter_override = false,
+            dataset_suffix = "hail",
+    }
+
+    call ExtractAvroFilesForHail.GvsExtractAvroFilesForHail {
+        input:
+            go = GvsQuickstartVcfIntegration.done,
+            project_id = project_id,
+            dataset = GvsQuickstartVcfIntegration.dataset_name,
+            filter_set_name = GvsQuickstartVcfIntegration.filter_set_name,
+            scatter_width = 10,
+    }
+
+    call CreateAndTieOutVds {
+        input:
+            branch_name = branch_name,
+            hail_wheel = hail_wheel,
+            avro_prefix = GvsExtractAvroFilesForHail.avro_prefix,
+            vds_destination_path = GvsExtractAvroFilesForHail.vds_output_path,
+            tieout_vcfs = GvsQuickstartVcfIntegration.output_vcfs,
+            tieout_vcf_indexes = GvsQuickstartVcfIntegration.output_vcf_indexes,
+    }
+
+    output {
+        Array[File] output_vcfs = GvsQuickstartVcfIntegration.output_vcfs
+        Array[File] output_vcf_indexes = GvsQuickstartVcfIntegration.output_vcf_indexes
+        Float total_vcfs_size_mb = GvsQuickstartVcfIntegration.total_vcfs_size_mb
+        File manifest = GvsQuickstartVcfIntegration.manifest
+        String vds_output_path = GvsExtractAvroFilesForHail.vds_output_path
+        Boolean done = true
+    }
+}
+
+
+task CreateAndTieOutVds {
+    input {
+        File hail_wheel
+        String branch_name
+        String avro_prefix
+        String vds_destination_path
+        Array[File] tieout_vcfs
+        Array[File] tieout_vcf_indexes
+    }
+    parameter_meta {
+        tieout_vcfs: {
+             localization_optional: true
+         }
+        tieout_vcf_indexes: {
+            localization_optional: true
+        }
+    }
+    command <<<
+        # Prepend date, time and pwd to xtrace log entries.
+        PS4='\D{+%F %T} \w $ '
+        set -o errexit -o nounset -o pipefail -o xtrace
+
+        script_url_prefix="https://raw.githubusercontent.com/broadinstitute/gatk/~{branch_name}/scripts/variantstore/wdl/extract"
+
+        for script in hail_gvs_import.py hail_join_vds_vcfs.py gvs_vds_tie_out.py
```

Comment:
this is clever, but it might be worth adding a comment somewhere about why we are doing this?